third_party/blink/renderer/modules/cobalt/h5vcc_experiments/h_5_vcc_experiments.cc

```diff
@@ -219,6 +218,26 @@ void H5vccExperiments::OnResetExperimentState(
   resolver->Resolve();
 }
 
+void H5vccExperiments::OnGetFeature(
+    ScriptPromiseResolver<V8OverrideState>* resolver,
+    h5vcc_experiments::mojom::blink::OverrideState feature_state) {
+  ongoing_requests_.erase(resolver);
+  switch (feature_state) {
+    case h5vcc_experiments::mojom::blink::OverrideState::OVERRIDE_USE_DEFAULT:
+      resolver->Resolve(V8OverrideState(V8OverrideState::Enum::kDEFAULT));
+      return;
+    case h5vcc_experiments::mojom::blink::OverrideState::
+        OVERRIDE_ENABLE_FEATURE:
+      resolver->Resolve(V8OverrideState(V8OverrideState::Enum::kENABLED));
+      return;
+    case h5vcc_experiments::mojom::blink::OverrideState::
+        OVERRIDE_DISABLE_FEATURE:
+      resolver->Resolve(V8OverrideState(V8OverrideState::Enum::kDISABLED));
+      return;
+  }
+  NOTREACHED() << "Invalid feature OverrideState for feature";
```

Comment:
Sholud this call `resolver->Reject` before calling NOTREACHED?

Reply:
Is it necessary since NOTREACHED is a fatal call?